deep_quoridor/src/utils/misc.py

```diff
@@ -116,3 +116,29 @@ def yargs(parser: argparse.ArgumentParser, default_path: str):
                     args_dict[k] = parser.parse_args(v["args"].split(" "))
 
     return args_dict
+
+
+def compute_elo(results: list["GameResult"], k=32, initial_rating=1500) -> dict[str, float]:
```

Comment:
Is `"GameResult"` a string in the type hint so that you don't need to import that class in this file?

Reply:
yes, because if I import the class I get a circular reference.  Using a string works anyway for the linter, so it's easier than having to re-organize things in the files to be able to import the lcass